oop/homework2.php

```diff
@@ -0,0 +1,65 @@
+<?php 
+abstract class Emailer 
+{
+  // Declare properties
```

Comment:
@ariga914 
correct this file to 1 tab = 4 spaces, I think you used 2 spaces only

Reply:
I've revised all the whitespaces.
Thanks for instructing me in a git hub's manner.
I'll check &click `Resolve conversation` next time.